src/containers/Home.jsx

```diff
@@ -91,7 +91,8 @@ const Content = props => {
                 target="_blank"
                 rel="noopener noreferrer"
               >
-                SXSE Podcast S01E03 - How To Make The Most Out Of Your First Co-op!
+                SXSE Podcast S01E03 - How To Make The Most Out Of Your First
+                Co-op!
```

Comment:
I'd say shove this back in with the previous line. 1 word doesn't need it's own line